test/functional/apps/dashboard_elements/input_control_vis/input_control_options.ts

```diff
@@ -28,6 +28,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   describe('input control options', () => {
     before(async () => {
       await visualize.initTests();
+      await timePicker.resetDefaultAbsoluteRangeViaUiSettings();
```

Comment:
`initTests` function is now selecting the default absolute time range, that's why we need to reset this value for this test.